tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyMain.java

```diff
@@ -3344,7 +3312,7 @@ private static void busted(String what, Throwable t, String stack) {
         if (t != null) {
             SurveyLog.logException(logger, t, what /* , ignore stack - fetched from exception */);
         }
-        logger.warning("SurveyTool " + SurveyMain.getCurrevCldrApps() + " busted: " + what + " ( after " + pages + "html+" + xpages
```

Comment:
Could fallback to utility or data hash here. This message might be in the logs out of context, and would provide information on which rev failed. 

Also, I would expect that cldr-apps would get a hash back at some point, so I'm not sure about ripping out support for a cldr-apps hash.

Reply:
I'll make it show the cldr-code hash here